doc.md

```diff
@@ -29,7 +29,13 @@ project_root
 [tool.pkgmt]
 github = "project/repository" 
 ```
-* Projects which do not contain a `setup.py` file. In this case the tool uses the `_version.py` file.
```

Comment:
I don't see entries to changelog.md please add. this will be a major version bump, since we're removing some functionality (new version: 0.7.0)